src/DigitalTwin/DigitalTwinUtility.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import * as vscode from 'vscode';
+import * as utils from '../utils';
+import { getExtension } from '../Models/Apis';
+import { ExtensionName } from '../Models/Interfaces/Api';
+import { DigitalTwinConstants } from './DigitalTwinConstants';
+
+/**
+ * Digital Twin extension utility
+ */
+export class DigitalTwinUtility {
+  // tslint:disable-next-line: no-any
+  private static instance: any;
```

Comment:
>private static instance: any; [](start = 2, length = 29)

An implication to use the APIs from DigitalTwinUtility is required to use DigitalTwinUtility.isAvailable() to check at first.
Can we initialize the "instance" by default rather than initialize it in the isAvailable() method?

The DigitalTwinUtility is more like a extension class for the DigitalTwins extension. how about using a more readable name for "instance" here?